### DISCUSSION TITLE
Can someone explain what the triangle with an exclamation mark icon represents in the iterm2 theme?

### DISCUSSION THREAD

**Discussion by ajsween** [JanDeDobbeleer/oh-my-posh#4151] [Themes]
Can someone explain what the triangle with an exclamation mark icon represents in the iterm2 theme?

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/9889883/c4c3b018-1674-4bc0-8d3b-3b6f8a018d72)

**Comment by JanDeDobbeleer** [comment]
@ajsween that's the [status](https://ohmyposh.dev/docs/segments/status) segment.
